src/containers/EncounterDetails/hooks.ts

```diff
@@ -21,11 +32,51 @@ export function useEncounterDetails(encounterId: string) {
             const encounter = sourceMap.Encounter[0];
             const patient = sourceMap.Patient[0];
             const practitioner = sourceMap.Practitioner[0];
-            const practitionerRole = sourceMap.PractitionerRole[0];
-            return { encounter, practitioner, practitionerRole, patient };
+
+            return {
+                id: encounter?.id,
+                patient,
+                practitioner: practitioner,
+                status: encounter?.status,
```

Comment:
Maybe there is no need to return `status` and `period` explicitly. Cause the hook returns the whole Encounter resource. And you ca get `status` from it where it's needed